.gitignore

```diff
@@ -24,4 +24,3 @@
 /public/bundles/
 /var/
 /vendor/
-###< symfony/framework-bundle ###
```

Comment:
this line needs to stay, this part is maintained by symfony

Reply:
done